neurodsp/sim/combined.py

```diff
@@ -82,3 +84,89 @@ def sim_combined(n_seconds, fs, components, component_variances=1):
     sig = np.sum(sig_components, axis=0)
 
     return sig
+
+
+@normalize
+def sim_central_freq(n_seconds, fs, central_freq, bw, ht, ap_func='sim_powerlaw',
+                     ap_kwargs={'exponent': -2}):
+    """
+    Returns a time series whose full power spectrum consists of an aperiodic component
+    and a gaussian peak at central_freq with standard deviation bw and relative height ht.
+
+
+    Parameters
+    -----------
+    n_seconds: float
+        Number of seconds elapsed in the time series.
+    fs: float
+        Sampling rate.
+    central_freq: float
```

Comment:
For consistency across periodic functions, I think this can / should just be called `freq`